public/Day 60/index.html

```
@@ -1,17 +1,18 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Travel Planner ✈️ | Plan Your Perfect Trip</title>
+    <meta name="description"
+        content="Plan and organize your trips with our interactive Travel Planner featuring itinerary management, maps, and budget tracking.">
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css">
```

HTMLHint (1.9.1)

[warning] The empty tag : [ link ] must be self closed.

(tag-self-close)